styles.css

```diff
@@ -508,6 +569,198 @@ button:active {
   }
 }
 
+/* Start the popup part */
+.workMainDiv {
+  display: flex;
+  background-color: #fff;
+  flex-direction: column;
+  height: auto;
+  width: 70%;
+  margin: 5vh 15%;
+  border-radius: 10px;
+}
+
+.workMainDiv span {
+  display: flex;
+  flex-direction: row-reverse;
+  font-size: 24px;
+  padding: 5px 10px;
+  color: rgb(29, 28, 28);
+}
+
+.workImgDiv {
+  display: flex;
+  flex-direction: column;
+  height: 54%;
+  width: 96%;
+  margin: 2%;
+  padding: 0;
+  border-radius: 5px;
+}
+
+.workImgDiv img {
+  height: 100%;
+  width: 100%;
+  border-radius: 10px;
+}
+
+.workContentDiv p {
+  justify-content: left;
+  font-size: 19px;
+  font-weight: normal;
+  line-height: 24px;
+  color: #344563;
+  padding: 5px 10px;
+}
+
+.workContentDiv {
+  display: flex;
+  flex-direction: column;
+  margin: 2%;
+}
+
+.workNameDiv {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Nice work making the change suggested by the previous reviewer :clap: However it will be nice to remove all the blank space to follow the Figma design
### Your design :point_down: 
![Screenshot from 2023-06-21 19-04-12](https://github.com/yusufyusie/yusufyusie.github.io/assets/91142432/4798252c-c413-4bc7-b956-03130692b980)
### Figma design :point_down: 
![Screenshot from 2023-06-21 19-32-24](https://github.com/yusufyusie/yusufyusie.github.io/assets/91142432/ad5755d3-89c4-4b00-b7d4-4df0f4a94d6d)